.github/workflows/CI.yml

```
@@ -74,15 +74,15 @@ jobs:
       - name: "Run tests ✅"
         shell: bash
         run: |
-          # this will compile the assets then run the tests
+          # this will compile the assets and translations then run the tests
           # check if there is a specific Sphinx version to test with
           # example substitution: tox run -e compile,py39-sphinx61-tests
           if [ -n "${{matrix.sphinx-version}}" ]; then
-            python -Im tox run -e compile,py$(echo ${{ matrix.python-version }} | tr -d .)-sphinx$(echo ${{ matrix.sphinx-version }} | tr -d .)-tests
+            python -Im tox run -e compile,i18n-compile,py$(echo ${{ matrix.python-version }} | tr -d .)-sphinx$(echo ${{ matrix.sphinx-version }} | tr -d .)-tests
```

Shouldn't this be:

```suggestion
            python -Im tox run -e compile-assets,i18n-compile,py$(echo ${{ matrix.python-version }} | tr -d .)-sphinx$(echo ${{ matrix.sphinx-version }} | tr -d .)-tests
```

Also, I'm a little confused about why you need to explicitly include compile-assets and i18n-compile because didn't you mark those as dependencies for the various test commands? I'm guessing there's something about tox here that I do not understand.

---

🤔 should have marked them as dependencies/required will double check

---

They are marked as `depends` in the tox file